osm2streets/src/transform/sausage_links.rs

```diff
@@ -137,24 +138,14 @@ fn fix(streets: &mut StreetNetwork, id1: RoadID, id2: RoadID) {
             lt: LaneType::Buffer(BufferType::Curb),
             dir: Direction::Fwd,
             width: LaneSpec::typical_lane_width(LaneType::Buffer(BufferType::Curb)),
+            turn_restrictions: Vec::new(),
         });
 
         for mut lane in road2.lane_specs_ltr {
             lane.dir = lane.dir.opposite();
             road1.lane_specs_ltr.push(lane);
         }
     }
-
-    // Tags
-    // TODO We shouldn't need to modify road1's tags; lanes_ltr are the source of truth. But...
```

Comment:
FWIW In A/B Street, I'm going to start explicitly storing the "current" and "original' lane specs. And at some point, I want to totally overhaul the `MapEdits` system there and bring it into this repo, possibly as an "add-on" crate. Representing edits (and persisting) to a `StreetNetwork` gets very complicated.